homeassistant/components/plugwise/gateway.py

```diff
@@ -44,7 +45,7 @@ async def async_setup_entry_gw(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     except InvalidAuthentication:
         LOGGER.error("Invalid username or Smile ID")
         return False
-    except PlugwiseException as err:
+    except (ClientConnectionError, PlugwiseException) as err:
```

Comment:
Longterm this should probably be caught by the library and raised as a library exception.

Reply:
Yeah, agree.

CC @CoMPaTech @bouwew

Reply:
Upstream issue: <https://github.com/plugwise/python-plugwise/issues/187>

Reply:
We've fixed this in plugwise-beta some time ago, in almost the same way.

We need to think about how to do this in the library...